src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,7 +1,88 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import ProductExplainSection from './ProductExplainSection';
+import ProductFloatContents from './ProductFloatContents';
+import ProductHeadSection from './ProductHeadSection';
+import ProductNotiSection from './ProductNotiSection';
+import ProductReviewSection from './ProductReviewSection';
 
 const ProductDetail = () => {
-  return <div>ProductDetail</div>;
+  const [product, setProduct] = useState({});
+  const [review, setReview] = useState({});
+  const [location, setLocation] = useState({});
+
+  useEffect(() => {
+    // console.log('componentDidMount');
+    fetch(`http://10.58.3.220:8000/product/1`)
+      .then(result => result.json())
+      .then(product => setProduct(product.result));
+  }, []);
+  useEffect(() => {
+    // console.log('componentDidMount');
+    fetch(`http://10.58.3.220:8000/product/1/review`)
+      .then(result => result.json())
+      .then(data => setReview(data.result));
+  }, []);
+  useEffect(() => {
+    // console.log('componentDidMount');
+    fetch(`http://10.58.5.201:8000/product/1`)
+      .then(result => result.json())
+      .then(location => setLocation(location.result));
+  }, []);
```

Comment:
```suggestion
useEffect(() => {
    // console.log('componentDidMount');
    fetch(`http://10.58.3.220:8000/product/1`)
      .then(result => result.json())
      .then(product => setProduct(product.result));

    fetch(`http://10.58.3.220:8000/product/1/review`)
      .then(result => result.json())
      .then(data => setReview(data.result));

    fetch(`http://10.58.5.201:8000/product/1`)
      .then(result => result.json())
      .then(location => setLocation(location.result));
  }, []);
```
하나의 useEffect 내에서 fetch 함수를 여러번 사용하셔도 됩니다!
그리고 첫번째 fetch와 마지막 fetch는 다른 데이터를 가져오나요?

Reply:
반영하였습니다.